cmd/lncli/main.go

```diff
@@ -2,28 +2,33 @@ package main
 
 import (
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
+	"time"
+
+	"gopkg.in/macaroon-bakery.v1/bakery/checkers"
+	"gopkg.in/macaroon.v1"
```

Comment:
> Unfortunately, macaroon-bakery actually imports itself, the macaroon package, and other libraries using gopkg.in

Le sigh...

Alright, for now let's do this: we'll merge this as is now, but then we'll fork the repo's and modify them to _not_ use `gopkg.in` in a follow up PR.